resources/zib/zib-PharmaceuticalProduct.xml

```diff
@@ -0,0 +1,176 @@
+  <purpose value="This Medication resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) PharmaceuticalProduct-v2.1.2(2020EN)](https://zibs.nl/wiki/PharmaceuticalProduct-v2.1.2(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-pharmaceuticalproduct-v2.1.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/PharmaceuticalProduct-v2.1.2(2020EN)" />
+    <name value="zib PharmaceuticalProduct-v2.1.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Medication" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Medication" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Medication">
+      <path value="Medication" />
+      <short value="PharmaceuticalProduct" />
+      <definition value="The prescribed product is usually a medicine. However, medical aids and bandages can also be prescribed and supplied via the pharmacy. Strictly speaking, food and blood products do not belong in the medication category, but can be prescribed and supplied by a pharmacy as well. &#xD;&#xA;&#xD;&#xA;A type of medication can be indicated with a **single code**. That code can be chosen from several possible coding systems (concretely: GPK, PRK, HPK or article numbers). Correct use of these codes in the software systems will sufficiently record the composition of the product used, making a complete product specification unnecessary. &#xD;&#xA;&#xD;&#xA;In addition to a primary code, **alternative codes** from other coding systems can also be entered (so that the GPK can be sent along in the event that the patient was registered based on PRK, for example). &#xD;&#xA;&#xD;&#xA;Entering multiple ingredients will enable you to display a compound product. In that case, the **composition of the medication** can be specified implicitly (with the use of a medication code) or explicitly, for example by listing the (active) substance(s) of the medication. &#xD;&#xA;&#xD;&#xA;**Prescriptions to be prepared by the pharmacy** can be entered as well. This can be done by means of the option listed above to enter coded ingredients and/or by entering the composition and preparation method as free text." />

```

Comment:
a **single code* must be **a single code**

Reply:
Why is description included here?

Reply:
@Ashwin-nictiz : adjusted for now.
@pieter-edelman-nictiz: because the zib also has this text on the root. I made a ticket: https://bits.nictiz.nl/browse/ZIB-1452

I think the zibs is going for having this text in one place.

Reply:
The zib centre has proposed to remove the text on the root definition. I aligned the profile with that proposal.

Reply:
I meant: the profiling guidelines state that the definition of the zib root element is excluded from metadata mapping, so why is it included in this case?

Reply:
Because it actually had some value. 
The profiling guidelines: "unless it's the root concept of the zib (because this will always be Root concept of the [...] information model ..., which doesn't provide much information)."

The 'because this will always be Root concept of the[...]' turned out not to be true.